packages/xchain-bitcoincash/src/utils.ts

```diff
@@ -192,9 +192,10 @@ export const parseTransaction = (tx: Transaction): Tx => {
  */
 export const validateAddress = (address: string, network: Network): boolean => {
   try {
-    bitcash.address.toOutputScript(toLegacyAddress(address), bchNetwork(network))
-    return true
+    const toAddress = toCashAddress(address)
+    return bchaddr.isValidAddress(toAddress) && bchaddr.detectAddressNetwork(toAddress) === network
   } catch (error) {
+    console.log(error)
```

Comment:
Still needed ^? Or maybe `console.warn`?

Reply:
yeah good catch...i think i'm going to remove the catch altogether..